net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checks/blocks/checkstyle-metadata.xml

```diff
@@ -4,165 +4,5 @@
 "http://eclipse-cs.sourceforge.net/dtds/checkstyle-metadata_1_1.dtd">
 <checkstyle-metadata>
     <rule-group-metadata name="%Blocks.group" priority="800">
-
-        <rule-metadata name="%AvoidNestedBlocks.name" internal-name="AvoidNestedBlocks" parent="TreeWalker">
-            <alternative-name internal-name="com.puppycrawl.tools.checkstyle.checks.blocks.AvoidNestedBlocksCheck"/>
-            <description>%AvoidNestedBlocks.desc</description>
-            <property-metadata name="allowInSwitchCase" datatype="Boolean" default-value="false">
-                <description>%AvoidNestedBlocks.allowInSwitchCase</description>
-            </property-metadata>
-            <quickfix classname="net.sf.eclipsecs.ui.quickfixes.blocks.AvoidNestedBlocksQuickfix"/>
```

Comment:
When reviewing the changes, I noticed this line.
Without this link in the XML file, does this mean quickfixes will be broken?

Reply:
Break goes beyond this change and release. See https://github.com/checkstyle/eclipse-cs/issues/315 .